.vscode/launch.json

```diff
@@ -1,27 +1,14 @@
 {
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
You and @Nowbob will have to agree on this, because I see this change in your PRs everytime, youre VSCode adds these comments, Nowbob's removes it ^^ it's no big deal so it won't spot me from merging but if we could have a proper handling on this, it'd be better 😄

Reply:
As I mentioned in my PR description, I reverted the state of the files under `.vscode` to the first commit I made that introduced them. After speaking with Bob about the issue, he acknowledged that his modifications to those files were unintentional while he was working on other issues. So I rolled the files back because as they were, they were broken.

VS Code is not mysteriously editing these files. It was all done by hand. I just want to reassure you that these changes are deterministic and intentional. I might clean up and remove the comment later. VS Code did generate that comment when the file was first created, but that's it. If anything I didn't remove it initially when I probably should have.